### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by mtyler**
This change worked locally for me.  My git-fu is sad and resulted in showing the changes to index.html.  This was probably because I used c/p instead of git-fu.  but anyway...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,411 +44,330 @@
...
+            </ol>
+          </section>
+
+          <section>
+            <h3>Compliance Automation</h3>
+            <ol>
+              <li>Compliance policies are expressed in code</li>
+              <li>Compliance is checked automatically</li>
+              <li>Compliance automation removes non-compliant nodes from production upon detection</li>
+              <li>Nodes are destroyed beyond a certain age (e.g. 30 dyas)</li>
```
Comment (irvingpop) [inline index.html]: Hey would you mind fixing this typo (dyas) and save me a PR?

**Comment by mtyler** [comment]
fixed the typo in the md file

**Comment by mtyler** [comment]
I've got some ideas around what we can produce in terms of reports, and I've introduced these ideas into the slides.
1. radar graph of current + desired scores
2. Report with specific scoring for each Q
3. Worksheet showing all Qs with deltas and space for filling in Strategic Objective and a space for filling in tactical objectives.

**Comment by mtyler** [comment]
I'll demo what I've done here but this branch includes a functional MVP for an automated Assessment Session >>> Customer Deliverable

**Comment by mtyler** [comment]
@irvingpop @justinredd a 👍  / 👎  would be great

**Comment by irvingpop** [comment]
hey @mtyler I'm noticing that now there's merge conflicts.  could you please resolve those?

**Comment by mtyler** [comment]
@irvingpop Thanks for pointing out.  Should be good now.

**Comment by irvingpop** [comment]
👍 great work @mtyler !!!

**Pull request merged**